util/quick_trig.cpp

```diff
@@ -40,9 +40,9 @@
  ******************************************************************************/
 
 
-#include "util/quick_trig.h"
+#include "util/quick_trig.hpp"
 
-#include <math.h>
+#include <cmath>
 #include "util/maths.h"
```

Comment:
done in https://github.com/lis-epfl/MAVRIC_Library/pull/326/commits/5a72d551741072722e02f1082bef304f3ab1d40b